hashintel/hash
# Add banners to repo READMEs

## 🌟 What is the purpose of this PR?

Add various cover images to the README, and shields in a couple of places they were missing.

## How to test

Visit and check the image/shields on the following pages:
1. https://github.com/hashintel/hash/blob/dw/misc-changes/packages/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/apps/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/blocks/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/infra/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/README.md
